vuepress/docs/next/tutorials/curate/ecr-private-images.md

```diff
@@ -8,10 +12,39 @@ The standard deployment of the Entando Component Repository assumes that plugin
 * A bundle containing a microservice plugin based on an image from a private repository. You can set this up by [creating a microservice bundle](../create/ms/generate-microservices-and-micro-frontends.md) and making the corresponding Docker Hub repository private.
 
 ## Tutorial
-The first step demontrates how to create a secret for Docker Hub but please see the [corresponding Kubernetes documentation](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry) for other options. Once you have the created the secret you can either apply it to a deployed Entando application or add it to the Helm template for a new deployment.
+The first step demontrates how to create a secret for Docker Hub but please see the [corresponding Kubernetes documentation](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry) for other options. Once you have the created the secret you can either apply it to a deployed Entando application or add it to the plugin descriptor file YOUR-PLUGIN.yaml for a new deployment. See the [Plugin Environment Variables](../devops/plugin-environment-variables.md) tutorial for detailed information.
 
-**1. Create the secret**
-Supply the following parameters:
+ **1. Create the secret**
+* Determine YOUR-BUNDLE-ID. You can do this from your project directory using this command:
+```
+ent prj get-bundle-id
+```
+it should look something like this:
+```
+$ent ecr https://github.com/YOUR-DIR/YOUR-APPLICAIN-GIT-REPO.git
+343826ca
+```
+* Create a Secret named YOUR-BUNDLE-ID-my-secret with a key-value pair mySecretKey=mySecretValue. Make sure to replace YOUR-BUNDLE-ID with the value from the previous step.
+```
+kubectl create secret generic YOUR-BUNDLE-ID-my-secret --from-literal=mySecretKey=mySecretValue -n entando
+```   
+Verify that the plugin file bundle/plugins/YOUR-PLUGIN.yaml specifies descriptorVersion: v4.
+Insert an environmentVariables section into YOUR-PLUGIN.yaml after replacing YOUR-BUNDLE-ID with the correct value. By convention, environment variables are all caps and K8s resource names are hyphenated.
+Here is an example.
```

Comment:
example:

consider backticking so things stand out like we discussed in standup?